src/style.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+.formBig {
+  margin: 20px auto;
+}
+
+.searchInputBig {
+  width: 100%;
+  padding: 23px 62px;
+  box-sizing: border-box;
+  background: url(images/Search.svg) 25px center no-repeat;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.searchInputBig::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
```

Comment:
Placeholder inherits font-family of it's parent, this property here is redundant
```suggestion
```